packages/client/src/v2-events/features/events/actions/correct/request/router.tsx

```diff
@@ -13,10 +13,15 @@ import React from 'react'
 import { Outlet } from 'react-router-dom'
 import * as RequestCorrection from '@client/v2-events/features/events/actions/correct/request'
 import { ROUTES } from '@client/v2-events/routes'
+import { Action } from '@client/v2-events/features/events/components/Action'
 
 export const router = {
   path: ROUTES.V2.EVENTS.REQUEST_CORRECTION.path,
-  element: <Outlet />,
+  element: (
+    <Action>
```

Comment:
This component now makes sure that both form data state and metadata state are properly initialised before the view opens